README.md

```diff
@@ -113,4 +113,62 @@ Rather than using a local kafka instance, this project can utilize an instance o
 There is a provided docker-compose file (docker-compose-confluent-cloud.yml) that passes the above environment variables into the container that gets created. Further, this file doesn't spin up a local kafka instance since it is not required.
 
 ### Note
-This has only been tested with Confluent Cloud but technically all SASL authenticated Kafka brokers can be reached using this method.
\ No newline at end of file
+This has only been tested with Confluent Cloud but technically all SASL authenticated Kafka brokers can be reached using this method.
+
+## GitHub Artifact Usage
+
+To use this library in another application, add the GitHub package URLs to the `repositories` section in `pom.xml` of the consumer application or in your local `~/.m2/settings.xml` file. Here is an example implementation of using the GitHub artifact in a consumer application:
```

Comment:
` GitHub package URLs` referring to the whole <settings> xml?

Reply:
Not exactly, the `GitHub package URLs` only refers to the URL where you want to pull the specified package from. This needs to be set properly in the `url` field of the xml configuration:

```xml
<settings>
  <profiles>
    <profile>
      <repositories>
        <repository>
          <url>https://maven.pkg.github.com/usdot-jpo-ode/jpo-sdw-depositor</url>
        </repository>
      </repositories>
    </profile>
  </profiles>
</settings>
```